prow/plugins/updateconfig/updateconfig.go

```diff
@@ -86,6 +86,13 @@ func update(gc githubClient, kc kubeClient, org, repo, commit, name, namespace s
 	}
 
 	for key, filename := range updates {
+		if filename == "" {
+			if _, ok := data[key]; ok {
+				delete(data, key)
```

Comment:
nit: The key can be deleted without checking that it exists. From the [go blog](https://blog.golang.org/go-maps-in-action):
> The delete function doesn't return anything, and will do nothing if the specified key doesn't exist.

Reply:
done